CQLEngine.js

```diff
@@ -30,6 +30,19 @@ class CQLEngine {
         '2.4.0'
     ];
 
+    /**
+     * List of supported CQL versions.
+     * @type {string[]}
+     */
+    static cqlVersion = [
+        '1.0',
+        '1.1',
+        '1.2',
+        '1.3',
+        '1.4',
+        '1.5'
+    ];
+
```

Comment:
I included a cqlVersion list along with engineVersions. Although it's not currently utilized in any class, I've retained it in case it's needed for future extensions or validation purposes.